package.json

```diff
@@ -28,7 +28,7 @@
     "husky": "8.0.2",
     "lint-prepush": "2.2.1",
     "lint-staged": "10.0.0",
-    "prettier": "2.8.0"
+    "prettier": "1.19.1"
```

Comment:
we had to downgrade prettier? 😮

Reply:
The base package was set to 2.8.0, but Minespace, Core and the common package were all 1.19.1.  Making sure it's all on the same version.